courses/utils.py

```diff
@@ -0,0 +1,251 @@
+"""
+Utilities for courses/certificates
+"""
+import logging
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured, PermissionDenied
+from edx_api.client import EdxApi
+from requests.exceptions import HTTPError
+from rest_framework.status import HTTP_404_NOT_FOUND
+from courses.models import CourseRun, CourseRunGrade, CourseRunCertificate
+from users.api import User, fetch_user
+
+logger = logging.getLogger(__name__)
+
+
+def _verify_configuration(override_grade=None, user=None):
+    """
+    Verify if required variables are setup properly and the command arguments are correct
+    """
+    if settings.OPENEDX_GRADES_API_TOKEN is None:
+        raise ImproperlyConfigured("OPENEDX_GRADES_API_TOKEN is not set")
+
+    if settings.OPENEDX_API_KEY is None:
+        raise ImproperlyConfigured("OPENEDX_API_KEY is not set")
+
+    # Grade override for all users for the course run. Not allowed.
+    if override_grade and not user:
```

Comment:
I personally feel the logic related to `override_grade` is not related to ` _verify_configuration`. either the name should be different or not be in `_verify_configuration`